src/components/ToolTip/ToolTip.jsx

```diff
@@ -108,15 +108,12 @@ const ToolTip = createClass({
 	components: {
 		Target: createClass({
 			displayName: 'ToolTip.Target',
-			propName: 'Target',
 		}),
 		Title: createClass({
 			displayName: 'ToolTip.Title',
-			propName: 'Title',
 		}),
 		Body: createClass({
 			displayName: 'ToolTip.Body',
-			propName: 'Body',
```

Comment:
The weird thing with these is that these wouldn't be props of `ToolTip`, they'd be props of a `ToolTip.FlyOut`, which would actually be the child component if this were factored correctly, but that depends on a refactor of `ContextMenu`.